root/search/components/InstrumentResults.js

```diff
@@ -21,17 +21,12 @@ function buildResult(result, index) {
   const score = result.score;
 
   return (
-    <tr className={loopParity(index)} data-score={score} key={instrument.id}>
-      <td>
-        <EntityLink entity={instrument} />
```

Comment:
InstrumentListEntry uses DescriptiveLink instead, but AFAICT that is exactly equivalent for instruments.